python/hail/methods/impex.py

```diff
@@ -541,6 +541,38 @@ def import_fam(path, quant_pheno=False, delimiter=r'\\s+', missing='NA'):
            path=oneof(str, listof(str)),
            max_count=int)
 def grep(regex, path, max_count=100):
+    """Searches given paths for all lines containing regex matches.
+
+        Examples
+        --------
+
+        Print all lines containing the string ``hello`` in *file.txt*:
+
+        >>> hl.grep('hello','data/file.txt')
+
+        Print all lines containing digits in *file1.txt* and *file2.txt*:
+
+        >>> hl.grep('\d', ['data/file1.txt','data/file2.txt'])
+
+        Notes
+        -----
+        :meth:`grep` mimics the basic functionality of Unix ``grep`` in
```

Comment:
I think this should be :func:\`.grep\` Check the syntax of what other functions use.